core/app/modules/kit-library/assets/js/pages/preview/preview.scss

```diff
@@ -0,0 +1,27 @@
+.e-kit-library__preview {
+	&-loader {
+		position: absolute;
+		top: 0;
+		left: 0;
+		width: 100%;
+		height: 100%;
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		font-size: type( size, '30' );
+		color: tints( 500 );
+		z-index: 0;
+	}
+
+	&-iframe {
+		border: none;
+		transition: $transition-hover;
+		box-shadow: shadow("2");
```

Comment:
```suggestion
		box-shadow: shadow( '2' );
```

Reply:
Seems like that most of the code that was written in SCSS does not have spaces between parenthesis